compiler/luci/pass/src/PropagateQuantParamPassInternal.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_CIRCLE_PROPAGATE_QUANT_PARAM_PASS_INTERNAL_H__
+#define __LUCI_CIRCLE_PROPAGATE_QUANT_PARAM_PASS_INTERNAL_H__
+
+#include <luci/IR/CircleNodes.h>
+#include <luci/IR/CircleNodeVisitor.h>
+
+namespace luci
+{
+
+//  Visitor to propagate quantization parameters
+struct PropagateQuantParam final : public luci::CircleNodeMutableVisitor<bool>
```

Comment:
As I understand, this `PropagateQuantParam` class purpose is to be used (only) in `PropagateQuantParamPass` class
but is exposed with `luci` namespace by this internal header just for the testing.
If so, it would be better not to expose by this header and do testing with `PropagateQuantParamPass` class not with `PropagateQuantParam` class. 

Reason: Exposing this class to `luci` namespace with a header may let others reuse this. There is no information not to. But I don't think this was your intention and purpose of this class. I'm not sure adding a something like a WARNING comment will do...

If you have difficulties chaning the test, please let me know... we may have to find another way...
- (I've recently noticed there already exist two or more Passes working in this way. It would be better to fix them too, but when you have some spare time, with low priority.)

Reply:
> If so, it would be better not to expose by this header and do testing with PropagateQuantParamPass class not with PropagateQuantParam class.

I found this pattern (`XXXInternal.h`) from our legacy code (FuseInstanceNormPass). So I thought the pattern is acceptable. I've changed the test pattern. PTAL